lamp.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
```

Comment:
In general, I'd recommend having some sort of internal python directory hierarchy. It makes it a lot easier to reuse your code pieces, even if it's just being able to refer to `import tools.myFoo`

Reply:
1. lol because lamps give you 🧞 .  Happy to pick a more conventional name though if this one's not intuitive. XD

2. scripts/ sounds good. 
3. Yeah that's the goal. the script is in Python to be testable and I imagine we'd want to migrate more sh scripts in `scripts/` to anything-but-bash. Not going to do it now since testability wasn't a requirement, but happy to file a followup ticket.